Frameworks/CoreFoundationAdditions/_NSCFTemporaryRootObject.mm

```diff
@@ -16,4 +16,8 @@
 #include "_NSCFTemporaryRootObject.h"
 
 @implementation _NSCFTemporaryRootObject
+- (void)doesNotRecognizeSelector:(SEL)selector {
+    // According to the reference platform documentation, this method must never return.
+    abort();
```

Comment:
We can't take a dependency on error reporting here, as it requires Foundation to be loaded.

---
In reply to: [71574472](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71574472) [](ancestors = 71574472)